e2e/deployment.go

```diff
@@ -0,0 +1,106 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	appsv1 "k8s.io/api/apps/v1"
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	deploymentutil "k8s.io/kubernetes/pkg/controller/deployment/util"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2elog "k8s.io/kubernetes/test/e2e/framework/log"
+)
+
+// execCommandInPodWithName run command in pods using podName.
+func execCommandInPodWithName(f *framework.Framework, c, pn, cn, ns string) (string, string, error) {
```

Comment:
It really helps to use variables/parameters with a descriptive name. `c`, `pn` and `cn` do not tell me anything. `ns` usually stands for namespace, so  that's fine.

Reply:
Sure, parameter naming can be improved :+1: